internal/domain/rayfile/file.go

```diff
@@ -0,0 +1,33 @@
+package rayfile
+
+type Rayfile struct {
+	Version   string     `toml:"version"`
+	Workspace *workspace `toml:"workspace"`
+}
+
+type workspace struct {
```

Comment:
I think this should be **public**, and for all included structs, use suffix `Def`, like `WorkspaceDef`